doc/TRANSLATING.md

```diff
@@ -87,6 +88,20 @@ Click on the "Save" button when you are satisfied with your translation.
 
 See [Transifex's documentation][3] for more information.
 
+### Glossary
+
+This glossary is intended to help explain some CDDA-specific terms and their
+etymology in order to help translations.
+
+* **Exodii**: The Exodii are a bunch of humans from another dimension.  When
+  the Blob invaded their world, they managed to acquire enough technology to
+  open portals of their own, and now they portal to worlds that have been
+  attacked by the Blob and try to rescue survivors.  Exodii is a horrible
+  mangling of "Exodus" - the word literally means leaving or going out and has
+  connotations of forced emigration and refugees.  The Exodii are the people of
+  an Exodus.  Except Exodus is a Greek word and "ii" to indicate a plural is a
```

Comment:
https://en.wiktionary.org/wiki/exodus tells that 'exodus' is a latin word with origins in greek word 'éxodos'
https://www.latin-is-simple.com/en/vocabulary/noun/9294/ gives singular and plural forms

Reply:
Thanks, I've rephrased.